Src/lib/drivers/uart/uart.h

```diff
@@ -60,7 +60,7 @@ uart_err_t uart_deinit(obj_uart_t* objP_this);
 
  @return return type of error or ok if work correctly
  */
-uart_err_t uart_send(obj_uart_t* objP_this, uint8_t* str);
+e_uart_err_t uart_send(obj_uart_t* objP_this, uint8_t* str);
```

Comment:
If you uses objP_this it should be objPL_this since this is local variable

Reply:
Done